src/ts/order.ts

```diff
@@ -56,6 +56,15 @@ export interface Order {
   partiallyFillable: boolean;
 }
 
+/**
+ * Marker address to indicate that an order is buying Ether.
+ *
+ * Note that this address is only has special meaning in the `buyToken` and will
+ * be treated as a ERC20 token address in the `sellToken` position, causing the
+ * settlement to revert.
+ */
+export const BUY_ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```

Comment:
Will there also be a sell eth address?

Reply:
No, because:

> Does the EVM allow you to just transfer people's eth using offline signature?

No it does not!

Theoretically, we could sell ETH from SC wallets (like the Safe lets you set ETH allowances), but that would only work in very specific cases and requires a lot of additional things (like SC orders for example) to be sorted out first. We could also explore EIP-1271 (SC signatures), but that has problems of its own.